libraries/RW/Postgres/postgres.py

```diff
@@ -79,6 +75,19 @@ def parse_metric_and_time(
         """
         psql_result = psql_result.split("Time: ")
         for item in range(len(psql_result)):
-            psql_result[item] = psql_result[item].replace("\n","")
+            psql_result[item] = psql_result[item].replace("\n", "")
         query_details = dict({"metric": psql_result[0], "time": psql_result[1]})
-        return query_details
\ No newline at end of file
+        return query_details
+
+    @staticmethod
+    def quote_query(query: str) -> str:
+        """Simple helper method to escape specific characters that cause issues in the pod exec passthrough
+
+        Args:
+            query (str): _description_
+
+        Returns:
+            str: _description_
+        """
+        query = query.replace('"', '\\\\\\"')
```

Comment:
the problem is with single quotes; maybe doubles are also an issue, but related to the queries we were attempting, we need single quotes escaped.